src/templates/gwfcore/gwfcore-batch.template.yaml

```diff
@@ -81,7 +84,7 @@ Parameters:
   DefaultCEMaxvCpus:
     Type: Number
     Description: Maximum number of CPUs in the default compute environment. Should be >= than MinCpus
-    Default: 100
+    Default: 3000
```

Comment:
Suggest setting this to 1000. Most accounts have an on-demand EC2 vCPU limit around this value. Setting it too high result in jobs stuck in runnable.

Reply:
Thanks @wleepang , I have made the change.